dockstore-webservice/src/main/java/io/dockstore/webservice/core/webhook/InstallationRepositoriesPayload.java

```diff
@@ -88,6 +89,13 @@ public void setRepositoriesRemoved(List<WebhookRepository> repositoriesRemoved)
         this.repositoriesRemoved = repositoriesRemoved;
     }
 
+    @Override
+    @JsonIgnore
+    @Schema(hidden = true) // excluding the 'repository' property because it's not needed for this payload
+    public WebhookRepository getRepository() {
+        return super.getRepository();
+    }
```

Comment:
The goal of this was to remove "repository" from the InstallationRepositoriesPayload schema since it's not needed, but this doesn't seem to have any affect on the resulting openapi.yaml. Thus this can probably be removed.